rollout/experiment.go

```diff
@@ -167,6 +171,15 @@ func (c *RolloutController) reconcileExperiments(roCtx *canaryContext) error {
 		roCtx.SetCurrentExperiment(currentEx)
 	}
 
+	if currentEx != nil {
+		switch currentEx.Status.Status {
+		case v1alpha1.AnalysisStatusInconclusive:
+			roCtx.PauseContext().AddPauseCondition(v1alpha1.PauseReasonInconclusiveExperiment)
+		case v1alpha1.AnalysisStatusError, v1alpha1.AnalysisStatusFailed:
+			roCtx.PauseContext().AddAbort()
```

Comment:
Do we still need the ExperimentFailed flag if we have an aborted field? The ExperimentFailed flag will only be set to true if the abort flag is true and I can't think of a reason to have that granularity

Reply:
I don't think we need it. I was going to ask you the same question.

Reply:
I'll remove it